src/Search.php

```diff
@@ -22,22 +23,41 @@
 use Mathematicator\SearchController\OEISController;
 use Mathematicator\SearchController\SequenceController;
 use Mathematicator\SearchController\TreeController;
+use Symfony\Component\Translation\Translator;
 use Tracy\Debugger;
 
 class Search
 {
 
-	/** @var Engine */
+	/**
+	 * @var Engine
+	 */
```

Comment:
@miloshavlicek codestyle should be `/** @var Engine */`.

Reply:
I know, that's written in the log, however I am not able to fix it automatically on localhost
`php temp/code-checker/code-checker --short-arrays --strict-types --fix --no-progress` and
`php temp/coding-standard/ecs check src --config temp/coding-standard/coding-standard-php71.yml` does not change anything and results as done ok.

Reply:
Ok I understand. And who has to fix the codestyle? I?

Reply:
I'll fix it. Give me a minute.